lib/validators/qrda_cat3_validator.rb

```diff
@@ -1,4 +1,3 @@
-require 'validators/qrda_cat1_validator'
```

Comment:
Both.  :smile: 

It caused errors because it was not a require_relative and therefore was being required from the project using Cypress for external validation.

We also already require that file in `includes_for_external_validation.rb` and Cypress was still green after removing it.

**tl;dr** Not sure why it was in there to begin with. Doesn't seem to be necessary.